lib/WorkflowMain.groovy

```diff
@@ -0,0 +1,107 @@
+//
+// This file holds several functions specific to the main.nf workflow in the nf-core/atacseq pipeline
+//
+
+class WorkflowMain {
+
+    //
+    // Citation string for pipeline
+    //
+    public static String citation(workflow) {
+        return "If you use ${workflow.manifest.name} for your analysis please cite:\n\n" +
+            "* The pipeline\n" +
+            "  https://doi.org/10.5281/zenodo.3240506\n\n" +
```

Comment:
This point to chipseq 2.0 with the name for the framework in the zenodo record
Could only find this record for atacseq: https://doi.org/10.5281/zenodo.2656367, which is 1.0, nothing for the other releases.
Any idea what happened there @nf-core/core

Reply:
nevermind, proper zenodo doi is in README: https://zenodo.org/record/6544493

Reply:
```suggestion
            "  https://doi.org/10.5281/zenodo.2634132\n\n" +
```

Reply:
Well spotted! Will be fixed in a incoming PR.

Reply:
but then, we still should get rid of https://zenodo.org/record/2656368 which is a duplicate of https://zenodo.org/record/2634133

Reply:
Ah, I had issues with this when I first released the pipeline. Had to create one manually which we don't need now.